encrypted-media/idlharness.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>Encrypted Media Extentions IDL test</title>
+    <link rel="help" href="https://w3c.github.io/encrypted-media/">
+
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+    <script src=/encrypted-media/util/fetch.js></script>
```

Comment:
I believe the test I copied from used XHR and I just thought fetch() would be simpler. But I am not sure it is available on all browsers, hence the polyfill.